### PR TITLE
Shows / hides links based on city agency

### PR DESCRIPTION
**Pull request by fionawhim**
Also includes new way of invalidating login sessions so that when this
deploys we can ensure that people get their sessions populated
correctly.

See CityOfBoston/iam#565

### CODE REVIEW

File: services-js/access-boston/src/server/services/SamlAuth.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ interface SamlAuthAssertion {
       userAccessToken?: string[];
       userMFARegistrationDate?: string[];
       isUserRegistered?: string[];
+      cobUserAgency?: string[];
```
Comment (jessicamarcus) [inline services-js/access-boston/src/server/services/SamlAuth.ts:35]:
```suggestion
      userCobAgency?: string[];
```

Comment (jessicamarcus) [reply services-js/access-boston/src/server/services/SamlAuth.ts:35]: ^^ just to be consistent with the existing naming (`userAccessToken` et al.)

**Review by jessicamarcus — CHANGES_REQUESTED** [review]

File: services-js/access-boston/src/server/services/SamlAuth.ts (hunk 1 above)
Comment (fionawhim) [reply services-js/access-boston/src/server/services/SamlAuth.ts:35]: Those names come from the IAM team’s setup, so they’re not in our control.

Comment (jessicamarcus) [reply services-js/access-boston/src/server/services/SamlAuth.ts:35]: Ah - bummer!

**Review by jessicamarcus — APPROVED** [review]

**Pull request merged**